scripts/lint.js

```diff
@@ -1,12 +1,17 @@
 const chalk = require('chalk');
 const baseConfig = require('eslint-config-sku');
```

Comment:
Might be worth renaming this to `eslintConfigSku`, then you can use `baseConfig` further down the file as we do currently. Plus, I feel like this is actually clearer anyway.